fetch.js

```diff
@@ -380,6 +377,7 @@ var runTasks = function () {
     let timeEnded = new Date();
     if (err) {
       log.error('Error during fetching of data sources.');
+      log.error(err.stack);
```

Comment:
Please remove this or change to a `console.error()`. The logging goes to Papertrail and we've had issues with large stacks trying to get sent to the service which will crash the process. Worst case if it's changed to `console.error()` we can log in and check the logs from the process locally.

Reply:
Ok